l10n_it_website_portal_fatturapa/controllers/main.py

```diff
@@ -38,6 +38,12 @@ def details_form_validate(self, data):
         )
         if data["electronic_invoice_subjected"]:
             data["electronic_invoice_obliged_subject"] = True
+            country_italy = request.env.ref("base.it", raise_if_not_found=False)
+            country_id = data.get("country_id", False)
+            if country_id and country_italy and int(country_id) != country_italy.id:
+                data["codice_destinatario"] = "XXXXXXX"
+            elif data.get("codice_destinatario") == "XXXXXXX":
+                data["codice_destinatario"] = "0000000"
```

Comment:
Queste modifiche sono necessarie per la migrazione alla `16.0`?
Se non è così, sarebbero da spostare in un commit a parte, o anche in una PR a parte, e ci sarebbe da aprire la issue di tracciamento come descritto in https://github.com/OCA/l10n-italy/wiki/Team-di-sviluppo#apertura-issue

Reply:
fatta issue #3644